js/models/LocalSettings.js

```diff
@@ -15,12 +32,22 @@ export default class extends Model {
   }
 
   defaults() {
+    let langDataObj;
+
+    try {
+      langDataObj = getTranslationLangByCode(navigator.language.replace('-', '_'));
```

Comment:
I am defaulting this to detect the user's language. If it maps to one that we have a translation for, then that lang will be used. Otherwise, we fall back to us english.